internal/env/env.go

```diff
@@ -34,3 +34,7 @@ func IsLocalDev() bool {
 func SpeakeasyRunLocation() string {
 	return os.Getenv("SPEAKEASY_RUN_LOCATION")
 }
+
+func IsConcurrencyLockDisabled() bool {
+	return os.Getenv("SPEAKEASY_CONCURRENCY_LOCK_DISABLED") == "true"
```

Comment:
Should we consider a global CLI flag (`--disable-cli-locking`) that is more discoverable versus "hidden" environment variable as the escape hatch?